### DISCUSSION TITLE
Is cluster support on schedule?

### DISCUSSION THREAD

**Discussion by ghost** [brainix/pottery#29] [Q&A] [answered]
I build a redis cluster, so I need a python client that support it.

**Comment by brainix** [comment, marked as answer]
Hi, @hisi-SmallFish.

Thanks for your interest, but I believe that cluster support is better handled in a lower-level library, and not within Pottery itself.  Have you looked at [redis-py-cluster](http://redis-py-cluster.readthedocs.io/en/master/)?

Thanks,
Raj
